posts/2017-05-24-serverless-v1.14.0.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Google Cloud Functions support added in Serverless v1.14 deploy your functions to the Google Cloud
```

Comment:
"Serverless v1.14 - Deploy to Google Cloud Functions & rollback function support added"

Title tweak suggestion

cc @worldsoup